src/core/Button/styles.css

```diff
@@ -59,49 +66,8 @@
 .circle {
     border-radius: 50%;
     padding: 0;
-    height: 36px;
     width: 36px;
 }
-.circle i {
-    margin: 0;
-    vertical-align: middle;
-}
-
-/* SIZES */
-.small {
-    font-size: 13px;
-}
-
-.small.material-icons {
-    line-height: 14px;
-    height: 14px;
-    width: 23px;
-}
-.small.material-icons:first-child {
-    margin-right: 11px;
-}
-
-.small.material-icons:last-child {
-    margin-left: 11px;
-}
-
-.button.circle {
-    height: 32px;
-    width: 32px;
-    line-height: 32px;
-}
-.button.circle i {
-    margin-top: -6px;
-}
-
-.button.circle.material-icons {
-    margin: 0;
-    line-height: 10px;
-}
-
-.medium span {
-    transform: translateY(1px);
-}
 
 .large {
```

Comment:
as we are removing sizes, let's remove this as well. It would just confuse anyone reading it what other sizes are available